lib/components/split-pane.js

```diff
@@ -53,9 +75,13 @@ export default class SplitPane extends PureComponent {
       const total = this.props.children.length;
       const count = new Array(total).fill(1 / total);
 
-      sizes = count;
       sizes_ = count;
     }
+    return sizes_;
+  }
+
+  onDrag(ev) {
+    let sizes_ = this.getSizes();
```

Comment:
Does this need to be run on every drag event?

Reply:
I guess it was already doing that. This just makes it accessible from elsewhere.

Reply:
Oh, never mind. Yep it was 👍

Reply:
`const` here too I think 😃